packages/nodes-base/nodes/Postgres/Postgres.node.ts

```diff
@@ -72,10 +72,38 @@ export class Postgres implements INodeType {
 					},
 				},
 				default: '',
-				placeholder: 'SELECT id, name FROM product WHERE id < 40',
+				placeholder: 'SELECT id, name FROM product WHERE qty > $1 AND price <= $2',
```

Comment:
would not abbreviate quantity
```suggestion
				placeholder: 'SELECT id, name FROM product WHERE quantity > $1 AND price <= $2',
```